model_bakery/baker.py

```diff
@@ -698,6 +698,8 @@ def _handle_generic_foreign_keys(self, instance: Model, attrs: Dict[str, Any]):
             content_type_field = data["content_type_field"]
             object_id_field = data["object_id_field"]
             value = data["value"]
+            if value is None:
+                continue
             if is_iterator(value):
                 value = next(value)
```

Comment:
These two conditionals should be switched to properly handle the case in which a `None` value is provided within an iterator

Reply:
Thank you for the comment! Would you mind providing a patch for that? I won't be able to contribute soon to handle it, but will happily review it in case this is affecting you!